src/components/common/dialogs/showQueueDialog.js

```diff
@@ -53,13 +53,19 @@ export const showQueueDialog = (ContentComponent, { imageSource, imageHeight, bu
     marginBottom: getDesignRelativeHeight(15),
     width: '100%',
     height: getDesignRelativeHeight(129, false),
+    alignItems: 'center',
   }
   const StylesWrappedContent = withStyles(styles)(ContentComponent)
+  const Image = imageSource
 
   showDialogWithData(store.getCurrentSnapshot(), {
     type: 'queue',
     isMinHeight: true,
-    image: <Image source={imageSource} style={imageStyle} resizeMode="contain" />,
+    image: (
+      <View style={imageStyle}>
```

Comment:
why not just use pass props to Image ?
e.g.

```javascript
<Image style={imageStyle} resizeMode="contain" />
```

Reply:
also, why resizeMode="contain"  was removed ? or it was moved to the imageStyle?

Reply:
@serdiukov-o-nordwhale because these are svgs I need to pass the styles to a parent view to avoid styling and centering problems.
same with resizeMode, it's a prop for imgs but doesn't do anything for svgs